modules/installation-configuration-parameters.adoc

```diff
@@ -1692,55 +1608,104 @@ ifdef::alibabacloud[]
 [id="installation-configuration-parameters-additional-alibaba_{context}"]
 == Additional Alibaba Cloud configuration parameters
 
-Additional Alibaba Cloud configuration parameters are described in the following table. The `alibabacloud` parameters are the configuration used when installing on Alibaba Cloud. The `defaultMachinePlatform` parameters are the default configuration used when installing on Alibaba Cloud for machine pools that do not define their own platform configuration.
+Additional Alibaba Cloud configuration parameters are described in the following table. 
+
+The `alibabacloud` parameters are the configuration used when installing on Alibaba Cloud. The `defaultMachinePlatform` parameters are the default configuration used when installing on Alibaba Cloud for machine pools that do not define their own platform configuration. 
+
+These parameters apply to both compute machines and control plane machines where specified.
+
+[NOTE]
+====
+If defined, the parameters `compute.platform.alibabacloud` and `controlPlane.platform.alibabacloud` will overwrite `platform.alibabacloud.defaultMachinePlatform` settings for compute machines and control plane machines respectively.
+====
 
 .Optional {alibaba} parameters
 [cols=".^2,.^3,.^5a",options="header"]
 |====
 |Parameter|Description|Values
 
+|`compute.platform.alibabacloud.imageID`
+|The imageID used to create the ECS instance. ImageID must belong to the same region as the cluster.
+|String.
+
+|`compute.platform.alibabacloud.instanceType`
+|InstanceType defines the ECS instance type. Example: `ecs.g6.large`
+|String.
+
+|`compute.platform.alibabacloud.systemDiskCategory`
+|Defines the category of the system disk. Examples: `cloud_efficiency`,`cloud_essd`
+|String.
+
+|`compute.platform.alibabacloud.systemDisksize`
+|Defines the size of the system disk in gibibytes (GiB).
+|Integer.
+
+|`compute.platform.alibabacloud.zones`
+|The list of availability zones that can be used. Examples: `cn-hangzhou-h`, `cn-hangzhou-j`
+|String list.
+
+|`controlPlane.platform.alibabacloud.imageID`
+|The imageID used to create the ECS instance. ImageID must belong to the same region as the cluster.
+|String.
+
+|`controlPlane.platform.alibabacloud.instanceType`
+|InstanceType defines the ECS instance type. Example: `ecs.g6.large`
```

Comment:
Suggest to replace `ecs.g6.large` with `ecs.g6.xlarge`, as control-plane nodes requires 4 vCPU and 16G virtual RAM as a minimum. 

FYI
https://www.alibabacloud.com/help/en/elastic-compute-service/latest/instance-family#g6
https://docs.openshift.com/container-platform/4.12/installing/installing_gcp/installing-gcp-customizations.html#installation-minimum-resource-requirements_installing-gcp-customizations

Reply:
Thank you so much! This change is made, @jianli-wei